### PR TITLE
fix(electron5): support

### PR DESCRIPTION
**Pull request by hristoterezov**

### CODE REVIEW

File: alwaysontop/render.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ class AlwaysOnTop extends EventEmitter {
      */
     get _alwaysOnTopWindowVideo() {
         if (!this._alwaysOnTopWindow || !this._alwaysOnTopWindow.document) {
-            return;
+            return undefined;
```
Comment (saghul) [inline alwaysontop/render.js]: This looks unnecessary.

File: alwaysontop/render.js [hunk 2]
```diff
@@ -304,19 +312,13 @@ class AlwaysOnTop extends EventEmitter {
...
-            }
-
             const cssPath = path.join(__dirname, './alwaysontop.css');
             const jsPath = path.join(__dirname, './alwaysontop.js');
 
             // Add the markup for the JS to manipulate and load the CSS.
             this._alwaysOnTopWindow.document.body.innerHTML = `
               <div id="react"></div>
-              <video autoplay="" id="video" style="transform: none;"></video>
+              <video autoplay="" id="video" style="transform: none;" muted="muted"></video>
```
Comment (saghul) [inline alwaysontop/render.js]: I don't think a value is necessary for the `muted` property

File: popupsconfig/main.js [hunk 3]
```diff
@@ -28,7 +28,9 @@ function initPopupsConfiguration(jitsiMeetWindow) {
             event.newGuest = new BrowserWindow(Object.assign(options, {
                 titleBarStyle: undefined,
                 webPreferences: {
-                    nodeIntegration: false
+                    nodeIntegration: false,
+                    contextIsolation: false,
```
Comment (saghul) [inline popupsconfig/main.js]: not sure if we alphabeticallty sort in this project, but here goes nothing

File: alwaysontop/render.js (hunk 1 above)
Comment (hristoterezov) [reply alwaysontop/render.js]: ESLint complained....

File: alwaysontop/render.js (hunk 2 above)
Comment (hristoterezov) [reply alwaysontop/render.js]: Will try without it.

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jitsi-meet-electron-utils",
-  "version": "0.6.8",
+  "version": "0.6.9",
```
Comment (saghul) [inline package.json]: Make it 1.0 already!

Comment (hristoterezov) [reply package.json]: We should! But let's wait for the next minor bump at least?

**Pull request merged**